pic-api/src/main/kotlin/com/mashup/pic/security/jwt/JwtTokenFilter.kt

```diff
@@ -0,0 +1,61 @@
+package com.mashup.pic.security.jwt
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.mashup.pic.security.authentication.JwtAuthentication
+import jakarta.servlet.FilterChain
+import jakarta.servlet.http.HttpServletRequest
+import jakarta.servlet.http.HttpServletResponse
+import org.springframework.http.HttpHeaders
+import org.springframework.security.authentication.BadCredentialsException
+import org.springframework.security.core.context.SecurityContextHolder
+import org.springframework.web.filter.OncePerRequestFilter
+
+class JwtTokenFilter(
+        private val jwtTokenUtil: JwtTokenUtil,
+        private val objectMapper: ObjectMapper
+) : OncePerRequestFilter() {
+
+    override fun doFilterInternal(request: HttpServletRequest, response: HttpServletResponse, filterChain: FilterChain) {
+        val authorizationHeader = extractAuthorizationHeader(request)
+
+        authorizationHeader?.let {
+            runCatching {
+                val token = extractToken(it)
+                setAuthentication(token)
+            }.onSuccess {
+                filterChain.doFilter(request, response)
```

Comment:
runCatching 본문에 그냥 둬도 괜찮을듯?

Reply:
엇 말을 이해 몬했어용 본문에 둔다는 게 무슨 뜻이죠.?

Reply:
setAuthentication(token) 밑에 작성할 것 같아서요
```suggestion
      runCatching {
          setAuthentication(extractToken(authorizationHeader))
          filterChain.doFilter(request, response)
      }
```